app-backup/holland-backup-example/holland-backup-example-1.0.14.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="Holland Example Plugin"
+HOMEPAGE="http://www.hollandbackup.org/"
+
+MY_P="${P%%-*}-${P##*-}"
+
+SRC_URI="http://hollandbackup.org/releases/stable/${PV%.*}/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+RDEPEND=""
+PDEPEND="~app-backup/holland-${PV}[${PYTHON_USEDEP}]"
```

Comment:
Please reverse this RDEP/PDEP with holland core. It's more logical that plugins really require core, while the core only pulls plugins for later install.